tests/integration/main_test.go

```diff
@@ -46,6 +50,35 @@ func TestIstioJwt(t *testing.T) {
 	}
 }
 
+func TestIstioJwtScopes(t *testing.T) {
```

Comment:
We have a lot of duplicated code and it might improve readbility if we have a function that will take care of setup, cleanUp and run a given JWT test suite as there is some common setup.